torch/_decomp/decompositions.py

```diff
@@ -430,6 +431,47 @@ def safe_softmax(self, dim, dtype=None):
     return torch.where(masked_rows, zeros, out)
 
 
+@register_decomposition(aten.masked_select)
```

Comment:
Are there any tests for this?

Reply:
I don't think so, would you be able to point me to where the decomp tests are? I try adding one